app/src/main/java/in/ac/iitb/gymkhana/iitbapp/fragments/MapFragment.java

```diff
@@ -47,7 +47,8 @@ public void onMapReady(GoogleMap gMap) {
             googleMap.getUiSettings().setMyLocationButtonEnabled(true);
             googleMap.getUiSettings().setZoomGesturesEnabled(true);
         } else {
-            requestPermissions(new String[]{Manifest.permission.ACCESS_FINE_LOCATION}, 0);
+            Toast toast = Toast.makeText(getActivity(), "Need Permission", Toast.LENGTH_SHORT);
```

Comment:
This is not necessary since MapFragment will only be launched if permissions are granted